Formula/cocogitto.rb

```diff
@@ -0,0 +1,22 @@
+class Cocogitto < Formula
+  desc "Conventional Commits toolbox"
+  homepage "https://github.com/cocogitto/cocogitto"
+  url "https://github.com/cocogitto/cocogitto/archive/refs/tags/5.2.0.tar.gz"
+  sha256 "99f9dee05597d7721f6d046dbfefba5cb8d1c4ae22ced415f724affb3a6bd0cc"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", *std_cargo_args
+    generate_completions_from_executable(bin/"cog", "generate-completions",
+                                         shells: [:bash, :zsh, :fish], base_name: "cog")
```

Comment:
```suggestion
                                         base_name: "cog")
```
All three Shells is the default.

Reply:
sure, thanks